### ISSUE TITLE
Ping, nmdc: invalid context in user command: "3&#124;"

### ISSUE THREAD

**Issue by Kcchouette** [direct-connect/go-dcpp#62]
```
# dcping ping dchub://volks.volksfusion.net:4111
2019/03/08 00:16:46 invalid context in user command: "3&#124;"
{"addr":["dchub://volks.volksfusion.net:4111"],"status":"error"}
```

**Comment by Kcchouette** [comment]
Another problem with invalid context, but here with `7;\r\n`
```
# dcping ping dchub://landover3.no-ip.info:1905
2019/03/08 00:19:15 invalid context in user command: "7;\r\n"
{"addr":["dchub://landover3.no-ip.info:1905"],"status":"error"}
```

**Comment by Kcchouette** [comment]
Fixed in v0.8.8
